### PR TITLE
Unit and editor fixes

### PR DESCRIPTION
**Pull request by phlax**
- set unit index when resurrecting
- use min of total and max of slice in checking results

### CODE REVIEW

File: pootle/apps/pootle_store/models.py [hunk 1]
```diff
@@ -873,6 +873,7 @@ def resurrect(self, is_fuzzy=False):
             self.state = UNTRANSLATED
 
         self.update_qualitychecks(keep_false_positives=True)
+        self.index = self.store.max_index() + 1
```
Comment (ta2-1) [inline pootle/apps/pootle_store/models.py]: Put resurrected in the end always?
I feel that it will need to be changed :/

Comment (phlax) [inline pootle/apps/pootle_store/models.py]: where would you suggest, it has no index while its obsolete?

Comment (ta2-1) [inline pootle/apps/pootle_store/models.py]: Ideally index should come from a file though I'm not 100% sure.

Comment (phlax) [inline pootle/apps/pootle_store/models.py]: im not sure if it works correctly - i need to test further - but i think the indeces get updated after resurrection. If not we might need to change the order in the update

**Pull request merged**